.github/workflows/go.yml

```diff
@@ -16,6 +16,7 @@ jobs:
     env:
       TEST_KUBECTL: yes
       TEST_MINIKUBE_ENABLED: yes
+      TEST_K8S_STATE: yes
```

Comment:
Also print TEST_K8S_STATE env variable in "API Test" step.

Reply:
Fixed